index.js

```diff
@@ -57,8 +61,33 @@ const engine = require('./tools/main'),
           if (stat.isFile()) {
             console.log("'%s' is a template to be processed.", filename);
             let ret = await engine(resource, template, resourceFile);
-            const filenameOut = path.resolve(process.cwd(), './generated_html/' + fileshorthtml);
-            fse.writeFile(filenameOut, ret.body, 'utf-8');
+            const filenameOut = path.resolve(process.cwd(), './generated_html/' + fileshorthtml),
+
+              dom = new JSDOM(ret.body);
+
+            for (let componentName in global.fullObj) {
+              let style = dom.window.document.createElement('link'),

```

Comment:
Very big method, please split it up a bit, you may be able to pull the content of the foreach into a nicely named method to make it clearer what its doing

Reply:
Refactored this